ts/packages/ts/src/stores/Configu.ts

```diff
@@ -9,8 +9,20 @@ export type ConfiguConfigStoreConfiguration = {
   source?: string;
 };
 
+const ConfiguConfigStoreApprovalQueueErrorName = 'ConfiguConfigStoreApprovalQueueError';
+class ConfiguConfigStoreApprovalQueueError extends Error {
+  queueUrl: string;
+  readonly name: string = ConfiguConfigStoreApprovalQueueErrorName;
+
+  constructor(message: string, queueUrl: string) {
+    super(message);
+    this.queueUrl = queueUrl;
+  }
+}
+
 export class ConfiguConfigStore extends ConfigStore {
   private client: Axios;
+  public static readonly approvalQueueErrorName = ConfiguConfigStoreApprovalQueueErrorName;
```

Comment:
dont see any point of that

Reply:
using `error instanceof ConfiguConfigStoreApprovalQueueError` is unreliable, it gives us `false` when used through the browser SDK in the SaaS. So instead we check for the error via the `name` property which we export outwards through this class property. If we export the error itself we'd need to change the browser/node SDK to also export the errors and we didn't wanna change those SDKs. So this is the best solution we found so far, if you have any better ideas or input, we'd love it hear it.